packages/store-ui/src/atoms/Overlay/stories/Overlay.stories.tsx

```diff
@@ -0,0 +1,85 @@
+import type { Story, Meta } from '@storybook/react'
+import React from 'react'
+
+import type { ComponentArgTypes } from '../../../typings/utils'
+import type { Props as OverlayProps } from '../Overlay'
+import Component from '../Overlay'
+import mdx from './Overlay.mdx'
+
+type ColorType = 'black' | 'green'
+type OverlayStoryProps = OverlayProps & { color: ColorType }
+
+const OverlayTemplate: Story<OverlayStoryProps> = ({
+  color = 'black',
+  ...props
+}) => {
+  const styleMap = {
+    green: {
+      backgroundColor: 'rgba(0, 255, 0, 0.5)',
+    },
+    black: {
+      backgroundColor: 'rgba(0, 0, 0, 0.5)',
+    },
+  }
+
+  return (
+    <div style={{ height: 'calc(100vh + 150px)' }}>
+      A content outside me
```

Comment:
Maybe?
```suggestion
      Content outside of me
```

Reply:
👀 I think `Content outside me` is better, no? https://dictionary.cambridge.org/pt/dicionario/ingles/outside?q=outside+of
Outside vs outside of in this case has no difference. Ain't an English expert, remember this. hahha